### PR TITLE
added logs

### PR DESCRIPTION
**Pull request by vasilevskayaem**
from bt or TCP recieved data
incoming or outgoing connection
peer id
data amount

---

This change is [Reviewable](https://reviewable.io/reviews/thaliproject/thali_cordovaplugin/1444)

### CODE REVIEW

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (f8790cc)

**Comment by ThaliTester** [comment]
Test (Fail) 91243454 build is completed (f8790cc)

See https://github.com/ThaliTester/TestResults/tree/91243454f8790cc_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (dcdb86b)

**Comment by ThaliTester** [comment]
Test (Fail) 91243454 build is completed (dcdb86b)

See https://github.com/ThaliTester/TestResults/tree/91243454dcdb86b_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (4a4ed7f)

**Comment by ThaliTester** [comment]
Test (Fail) 91243454 build is completed (4a4ed7f)

See https://github.com/ThaliTester/TestResults/tree/912434544a4ed7f_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (4266744)

**Comment by ThaliTester** [comment]
Test (Success) 91243454 build is completed (4266744)

See https://github.com/ThaliTester/TestResults/tree/912434544266744_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
Test 912434544266744(4266744) has failed

See https://github.com/ThaliTester/TestResults/tree/912434544266744_added_logs_vasilevskayaem/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (533f962)

**Comment by ThaliTester** [comment]
Test (Success) 91243454 build is completed (533f962)

See https://github.com/ThaliTester/TestResults/tree/91243454533f962_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
Test 91243454533f962(533f962) has failed

See https://github.com/ThaliTester/TestResults/tree/91243454533f962_added_logs_vasilevskayaem/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (56e7290)

**Comment by ThaliTester** [comment]
Test (Success) 91243454 build is completed (56e7290)

See https://github.com/ThaliTester/TestResults/tree/9124345456e7290_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
Test 9124345456e7290(56e7290) has failed

See https://github.com/ThaliTester/TestResults/tree/9124345456e7290_added_logs_vasilevskayaem/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (fba6e27)

**Comment by ThaliTester** [comment]
Test (Success) 91243454 build is completed (fba6e27)

See https://github.com/ThaliTester/TestResults/tree/91243454fba6e27_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
Test 91243454fba6e27(fba6e27) has failed

See https://github.com/ThaliTester/TestResults/tree/91243454fba6e27_added_logs_vasilevskayaem/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (5012994)

**Comment by ThaliTester** [comment]
Test (Fail) 91243454 build is completed (5012994)

See https://github.com/ThaliTester/TestResults/tree/912434545012994_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (e97f7f5)

**Comment by ThaliTester** [comment]
Test (Success) 91243454 build is completed (e97f7f5)

See https://github.com/ThaliTester/TestResults/tree/91243454e97f7f5_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
Test 91243454e97f7f5(e97f7f5) has failed

See https://github.com/ThaliTester/TestResults/tree/91243454e97f7f5_added_logs_vasilevskayaem/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (8f3f6c6)

**Comment by ThaliTester** [comment]
Test (Success) 91243454 build is completed (8f3f6c6)

See https://github.com/ThaliTester/TestResults/tree/912434548f3f6c6_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
Test 912434548f3f6c6(8f3f6c6) has failed

See https://github.com/ThaliTester/TestResults/tree/912434548f3f6c6_added_logs_vasilevskayaem/ for the fail logs

**Comment by yaronyg** [comment]
This thing is 20 days old and hasn't been touched in 15 days. It didn't have anyone assigned to it. @vasilevskayaem what do you want us to do with this PR?

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (33cdf63)

**Comment by ThaliTester** [comment]
Test 91243454 (33cdf63) build started.

**Comment by ThaliTester** [comment]
Test (Fail) 91243454 build is completed (33cdf63)

See https://github.com/ThaliTester/TestResults/tree/9124345433cdf63_added_logs_vasilevskayaem/ for the logs

File: build.sh [hunk 1]
```diff
@@ -111,4 +111,4 @@ if [ $RUN_IN_CI == 0 ]; then
   # TODO Temporarily disabling ios build
   #rm -rf ThaliTest.app;ERROR_ABORT
   #cp -R ../ThaliTest/platforms/ios/build/device/ThaliTest.app ThaliTest.app;ERROR_ABORT
-fi
+fi
```
Comment (larryonoff) [inline build.sh]: please remove this change

File: src/android/java/io/jxcore/node/StreamCopyingThread.java [hunk 2]
```diff
@@ -141,6 +143,9 @@ public void run() {
                 totalNumberOfBytesWritten += numberOfBytesRead;
 
                 if (mNotifyStreamCopyingProgress) {
+//                     Log.v(TAG, mThreadName+ " " + "recieved " + numberOfBytesRead + " bytes from " +
```
Comment (larryonoff) [inline src/android/java/io/jxcore/node/StreamCopyingThread.java]: why this line is just commented ?

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 4. task. (657f94a)

**Comment by vasilevskayaem** [comment]
Review status: 0 of 5 files reviewed at latest revision, 2 unresolved discussions.

---

*[build.sh, line 114 at r1](https://reviewable.io:443/reviews/thaliproject/thali_cordovaplugin/1444#-KVUV2r35zzdScCpnOBC-r1-114:-KYIpNFaCVXXazEmB4PT:bk80jo9) ([raw file](https://github.com/thaliproject/thali_cordovaplugin/blob/33cdf634fbe6742d6e760bf9a0550586b2bae480/build.sh#L114)):*
<details><summary><i>Previously, larryonoff (larryonoff) wrote…</i></summary><blockquote>

please remove this change
</blockquote></details>

Are you sure, original master branch contains this line https://github.com/thaliproject/Thali_CordovaPlugin/blob/master/build.sh#L114

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/thaliproject/thali_cordovaplugin/1444)*

File: build.sh (hunk 1 above)
Comment (larryonoff) [inline build.sh]: @vasilevskayaem you shouldn't remove this __line__, but remove your __change__, since it's unnecessary.

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 4. task. (b94a005)

File: src/android/java/io/jxcore/node/StreamCopyingThread.java (hunk 2 above)
Comment (vasilevskayaem) [inline src/android/java/io/jxcore/node/StreamCopyingThread.java]: this pr fails on ci , I thought that probably logs create too mach spam and thats why ci fails, uncommented it for now

**Comment by ThaliTester** [comment]
Test 91243454 (657f94a) build started.

**Comment by ThaliTester** [comment]
Test 91243454 (b94a005) build started.

**Comment by ThaliTester** [comment]
Test (Fail) 91243454 build is completed (657f94a)

See https://github.com/ThaliTester/TestResults/tree/91243454657f94a_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
Test (Success) 91243454 build is completed (b94a005)

See https://github.com/ThaliTester/TestResults/tree/91243454b94a005_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
Test 91243454b94a005(b94a005) has failed

See https://github.com/ThaliTester/TestResults/tree/91243454b94a005_added_logs_vasilevskayaem/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (5653c00)

**Comment by ThaliTester** [comment]
Test 91243454 (5653c00) build started.

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (7c1b96c)

**Comment by ThaliTester** [comment]
Test 91243454 (7c1b96c) build started.

**Comment by ThaliTester** [comment]
Test (Fail) 91243454 build is completed (5653c00)

See https://github.com/ThaliTester/TestResults/tree/912434545653c00_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
Test (Success) 91243454 build is completed (7c1b96c)

See https://github.com/ThaliTester/TestResults/tree/912434547c1b96c_added_logs_vasilevskayaem/ for the logs

**Comment by ThaliTester** [comment]
Test 912434547c1b96c(7c1b96c) has failed

See https://github.com/ThaliTester/TestResults/tree/912434547c1b96c_added_logs_vasilevskayaem/ for the fail logs

**Comment by evabishchevich** [comment]
:lgtm:

---

Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/thaliproject/thali_cordovaplugin/1444#-:-KYrdfKCnQBY6b8RliKO:bnfp4nl)*

**Comment by larryonoff** [comment]
:lgtm:

---

Review status: 0 of 4 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/thaliproject/thali_cordovaplugin/1444#-:-KYsVLM4j9RHgaYEHLoN:bnfp4nl)*

**Comment by evabishchevich** [comment]
@larryonoff  asked me to remove "run CI" commit but this branch has a very strange history. Looks like it was rebased somewhere in past

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (47e35d1)

**Comment by ThaliTester** [comment]
Test 91243454 (47e35d1) build started.

**Comment by ThaliTester** [comment]
Test (Fail) 91243454 build is completed (47e35d1)

See https://github.com/ThaliTester/TestResults/tree/9124345447e35d1_added_logs_vasilevskayaem/ for the logs

**Comment by evabishchevich** [comment]
Test failed because of SIGSEGV. Commented in #1543. Everything works fine on my machine

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (4e24a36)

**Comment by ThaliTester** [comment]
Test 91243454 (4e24a36) build started.

**Comment by ThaliTester** [comment]
Test (Success) 91243454 build is completed (4e24a36)

See https://github.com/ThaliTester/TestResults/tree/912434544e24a36_added_logs_vasilevskayaem/ for the logs

**Comment by evabishchevich** [comment]
@yaronyg 2nd time build has completed successfully so I merge this PR

**Comment by ThaliTester** [comment]
Test 912434544e24a36(4e24a36) has failed

See https://github.com/ThaliTester/TestResults/tree/912434544e24a36_added_logs_vasilevskayaem/ for the fail logs

**Pull request merged**